### PR TITLE
chore: Preparing for 3.8.0 release

### PR DESCRIPTION
**Pull request by Mat001**
Summary
-------

-  decide api - release info: changelog, version bump, py version bump

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+
+### New Features
+* New Features
+Introducing a new primary interface for retrieving feature flag status, configuration and associated experiment decisions for users ([#309](https://github.com/optimizely/python-sdk/pull/309)). The new OptimizelyUserContext class is instantiated with createUserContext and exposes the following APIs to get OptimizelyDecision:
+
+    - setAttribute
+    - decide
+    - decideAll
+    - decideForKeys
+    - trackEvent
```
Comment (jaeopt) [inline CHANGELOG.md:14]: Change all these api names to python ones (snake-case)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Optimizely Python SDK Changelog
 
+## 3.8.0
+February 11th, 2021
```
Comment (jaeopt) [inline CHANGELOG.md:4]: Change it to 12th if cannot release it today

**Review by jaeopt — CHANGES_REQUESTED** [review]
Need changes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+
+### New Features
+* New Features
+Introducing a new primary interface for retrieving feature flag status, configuration and associated experiment decisions for users ([#309](https://github.com/optimizely/python-sdk/pull/309)). The new OptimizelyUserContext class is instantiated with createUserContext and exposes the following APIs to get OptimizelyDecision:
```
Comment (jaeopt) [inline CHANGELOG.md:8]:
```suggestion
Introducing a new primary interface for retrieving feature flag status, configuration and associated experiment decisions for users ([#309](https://github.com/optimizely/python-sdk/pull/309)). The new `OptimizelyUserContext` class is instantiated with `create_user_context` and exposes the following APIs to get `OptimizelyDecision`:
```

Comment (Mat001) [reply CHANGELOG.md:8]: done

File: CHANGELOG.md (hunk 1 above)
Comment (Mat001) [reply CHANGELOG.md:14]: done

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**